src/render/debug.rs

```diff
@@ -52,20 +55,31 @@ pub enum DebugShape {
     },
 }
 
+bitflags! {
+    #[derive(Default, Debug, Copy, Clone, PartialEq)]
+    pub struct DebugDrawFlags: u32 {
+        const DRAW_NORMAL = (1 << 0);
+        const DRAW_PICK = (1 << 1);
+    }
+}
+
```

Comment:
I don't think this is necessary. If an entity is passed it's implied that it draws to the pickbuffer

Reply:
I guess lines are added separately... I was mostly thinking of avoiding lines with this. But also the circles on the ruler. For ruler I'm using a rectangle that's a bit bigger for the pick buffer so it's clickable.